### PR TITLE
Make max concurrency parameter for number of hdfs scanners configurable and upgrade OSS sdk version

### PR DESCRIPTION
**Pull request by miomiocat**
#1336

### CODE REVIEW

File: be/src/common/config.h [hunk 1]
```diff
@@ -201,6 +201,8 @@ CONF_mInt32(doris_scan_range_row_count, "524288");
 CONF_mInt32(doris_scanner_queue_size, "1024");
 // single read execute fragment row size
 CONF_mInt32(doris_scanner_row_num, "16384");
+// number of max hdfs scanners
+CONF_Int32(doris_max_hdfs_scanner_num, "50");
```
[the remark this answers is not in the document]
Comment (miomiocat) [reply be/src/common/config.h:205]: ok, how about the remaining 7 variables prefixed with `doris_`? I can revise them together in this PR.

[the remark this answers is not in the document]
Comment (imay) [reply be/src/common/config.h:205]: Better to keep it for back compatibility.

[the remark this answers is not in the document]
Comment (miomiocat) [reply be/src/common/config.h]: done, PTAL

**Review by imay — DISMISSED** [review]

**Comment by imay** [comment]
run starrocks_clang-format

**Comment by imay** [comment]
run starrocks_clang-format

**Comment by imay** [comment]
@miomiocat should run ./build-support/clang-format.sh to make c++ code apply to the code style.

**Review by imay — APPROVED** [review]

**Review by Astralidea — APPROVED** [review]

**Pull request merged**